pkgs/applications/version-management/gitlab/default.nix

```diff
@@ -28,6 +28,10 @@ let
           patches = [ ./fix-grpc-ar.patch ];
           dontBuild = false;
         };
+        # the openssl needs the openssl include files
```

Comment:
Should this go in the commit with the manual changes?

Reply:
I don't see why; it's a required dependency for building the packages, as is everything else in the first commit.  The second commit adds the necessary changes to the NixOS module.